tools/azure-sdk-tools/devtools_testutils/azure_recorded_testcase.py

```diff
@@ -62,7 +64,7 @@ def add_sanitizer(sanitizer, **kwargs):
             json={
                 "regex": request_args["regex"],
                 "value": request_args["value"],
-                "groupForReplace": request_args["group_for_replace"]
+                "groupForReplace": request_args["group_for_replace"],
```

Comment:
Need to remove the extra `,`

Reply:
This was actually added by `black` -- I think it's a style preference and shouldn't affect functionality